assets/pages/new-contributions-landing/components/ContributionForm.jsx

```diff
@@ -236,84 +187,20 @@ function ContributionForm(props: PropTypes) {
-            name="contribution-email"
-            label="Email address"
-            value={email}
-            type="email"
-            autoComplete="email"
-            placeholder="example@domain.com"
-            icon={<SvgEnvelope />}
-            onInput={props.updateEmail}
-            isValid={checkEmail(email)}
-            pattern={emailRegexPattern}
-            formHasBeenSubmitted={checkoutFormHasBeenSubmitted}
-            errorMessage="Please provide a valid email address"
-            required
-            disabled={isSignedIn}
-          />
-          <Signout isSignedIn={isSignedIn} />
-          <NewContributionTextInput
-            id="contributionFirstName"
-            name="contribution-fname"
-            label="First name"
-            value={firstName}
-            icon={<SvgUser />}
-            autoComplete="given-name"
-            autoCapitalize="words"
-            onInput={props.updateFirstName}
-            isValid={checkFirstName(firstName)}
-            formHasBeenSubmitted={checkoutFormHasBeenSubmitted}
-            errorMessage="Please provide your first name"
-            required
-          />
-          <NewContributionTextInput
-            id="contributionLastName"
-            name="contribution-lname"
-            label="Last name"
-            value={lastName}
-            icon={<SvgUser />}
-            autoComplete="family-name"
-            autoCapitalize="words"
-            onInput={props.updateLastName}
-            isValid={checkLastName(lastName)}
-            formHasBeenSubmitted={checkoutFormHasBeenSubmitted}
-            errorMessage="Please provide your last name"
-            required
-          />
-          <NewContributionState
-            onChange={props.updateState}
-            selectedState={state}
-            isValid={checkState(state)}
-            formHasBeenSubmitted={checkoutFormHasBeenSubmitted}
-            errorMessage="Please provide a state"
-          />
-          <NewContributionPayment onPaymentAuthorisation={onPaymentAuthorisation} />
-          <PaymentFailureMessage checkoutFailureReason={props.paymentError} />
-          <NewContributionSubmit
-            whenUnableToOpen={props.setCheckoutFormHasBeenSubmitted}
-          />
-          {props.isWaiting ? <ProgressMessage message={['Processing transaction', 'Please wait']} /> : null}
-        </form>
-        <DirectDebitPopUpForm
-          onPaymentAuthorisation={onPaymentAuthorisation}
-          isPopUpOpen={props.isDirectDebitPopUpOpen}
-        />
-      </div>
-    );
+  return (
+    <form onSubmit={onSubmit(props)} className={classNameWithModifiers('form', ['contribution'])} noValidate>
+      <NewContributionType />
+      <NewContributionAmount
+        countryGroupDetails={props.selectedCountryGroupDetails}
+        checkOtherAmount={checkOtherAmount}
+      />
+      <ContributionFormFields />
+      <NewContributionPayment onPaymentAuthorisation={onPaymentAuthorisation} />
```

Comment:
while we're at it, let's give this a more helpful name: `PaymentMethodRadioButtons` or `PaymentMethodSelector` or similar